plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss

```diff
@@ -92,12 +92,12 @@
   }
 }
 
-.feature-content .x-panel-header .x-btn-drilldown-medium {
+.nx-feature-content .x-panel-header .x-btn-nx-drilldown-medium {
```

Comment:
Yes, it’s an extjs thing.

The value of 'ui' propagates to child classes. In the above example, 'ui' has been set to 'nx-drilldown', which turns .x-btn-{ui}-drilldown-medium into .x-btn-nx-drilldown-medium.